general/development/tools/behat.md

```diff
@@ -0,0 +1,721 @@
+| chromedriver | Use Chrome with Chromedriver directly | No | Yes |
+| headlesschromedriver | Use Chrome with Chromedriver directly, without displaying the GUI | No | No |
+| edge | Use Edge via Selenium | Yes | Yes |
+| headlessedge | Use Edge via Selenium, without displaying the GUI | Yes | No |
+| edgedriver | Use Edge with Edgedriver directly | No | Yes |
+| headlessedgedriver | Use Edge with Edgedriver directly, without displaying the GUI | No | No |
+
+#### chromedriver-wrapper
+
+When using Google Chrome, you must use the correct version of the `chromedriver` browser driver for the version of Chrome that you use.
+
+Since Google Chrome automatically updates on a regular basis, you will need to regularly upgrade your driver to match the version of Chrome that you are using.
+
+To make this easier, a `chromedriver-wrapper` utility has been written. This inspects the version of Chrome that is in your path, and downloads the correct version of `chromedriver` for that version, before starting it.
+
+Installation instructions can be found at [https://github.com/andrewnicols/chromedriver-wrapper](https://github.com/andrewnicols/chromedriver-wrapper).
+
+## Getting started (basic)
+
+This is a quick walk through to get Behat running for the first time.
+
+### Setting up
+
+There are a number of ways of configuring Behat on Moodle. This is one of the simplest.
+
+These notes assume that you have already installed a supported Java Runtime Environment, and the [moodle-browser-config](/general/development/tools/behat#pre-configured-browser-profiles-moodle-browser-config) tool.
+
+#### Setting up Selenium
+
+Generally we recommend use of Selenium, though this is not a fixed requirement. You can use the browser's driver implementation directly but this is harder to setup for the first time.
+
+Selenium is written in Java, and requires a recent version of the JRE to run. Please ensure that you have this installed prior to starting.
+
+Since Moodle 3.9.5 / 3.10.2 / 3.11.0 Moodle will work with any modern version of Selenium. At time of writing that is `3.141.59`, and 4.1.0.
+
+**Note:** There appears to be an issue with the version 4 jars - particularly in 4.1.0 that generates a **"Could not open connection: Capability value must be set"** error which appears to be an issue between Selenium and the MInkExtension for JavaScript tests - see https://githubmemory.com/repo/Behat/MinkExtension/activity for more information. The quick work around is to use the earlier version (3.141.59).
+
+1. Download the Selenium Server (Grid) from [https://www.selenium.dev/downloads/](https://www.selenium.dev/downloads/). This is a single JAR file, put it anywhere handy.
+1. Start Selenium
+
+   ```
+      # Version 3.141.59:
+
+         $ java -jar selenium-server-standalone-3.141.59.jar
+
+      # Version 4.0.0 and later:
+
+         $ java -jar selenium-server-4.0.0-beta-3.jar standalone
+   ```
```

Comment:
```suggestion
   ```console tile="Version 3.141.59"
java -jar selenium-server-standalone-3.141.59.jar
   ```

   ```console tile="Version 4.0.0 and later"
java -jar selenium-server-4.0.0-beta-3.jar standalone
   ```
```

Reply:
It would be great to add the `console` type to the rest of the code and remove $ from the beginning (it will help users to copy&paste it).